packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -12,6 +12,7 @@ import { IconButton } from '../../Button';
 import { IBreadcrumbProps, IBreadcrumbItem } from './Breadcrumb.Props';
 import { DirectionalHint } from '../../common/DirectionalHint';
 import { ResizeGroup } from '../../ResizeGroup';
+import { TooltipHost, TooltipOverflowMode } from '../../Tooltip';
```

Comment:
This adds tooltip to breadcrumb graph, increasing the weight of it. It makes me nervous. We really need a build system which spits out the bundle differences here, so that you can be nervous too. For example, if you make a bundle that just bundles the Breadcrumb, you would see that this probably adds 5k of stuff. Then we could ask how we reduce that or defer it, because tooltips are not needed at glass render.